include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -189,6 +196,12 @@ ORT_API_STATUS(OrtCreateEnvWithCustomLogger, OrtLoggingFunction logging_function
 ORT_API_STATUS(OrtCreateSession, _In_ OrtEnv* env, _In_ const ORTCHAR_T* model_path,
                _In_ const OrtSessionOptions* options, _Out_ OrtSession** out);
 
+/**
+ * \create a session from opaque model handle.
+ */
+ORT_API_STATUS(OrtCreateSessionFromHandle, _In_ OrtEnv* env, _In_ OrtModel* model_handle,
```

Comment:
How about 'OrtCreateSessionFromModel'? I think 'Handle' is too generic for the name as the type is OrtModel.